buildScripts/release.groovy

```diff
@@ -35,7 +36,19 @@ pipeline {
                         git config --global user.email "microprofile-bot@eclipse.org"
                         git config --global user.name "Eclipse MicroProfile bot"
                     '''
-                    sh "mvn -s /home/jenkins/.m2/settings.xml release:prepare release:perform -B -Dtag=${params.tag} -DdevelopmentVersion=${params.snapshotVersion} -DreleaseVersion=${params.releaseVersion} -Drevremark=${params.revremark}"
+
+                    def settingsXml = '/home/jenkins/.m2/settings.xml'
+
+                    if (params.stagingList != '') {
+                        sh '''
+                            wget https://github.com/xstefank/staging-augmenter/raw/main/staging-augmenter
```

Comment:
Some systems don't have `wget` by default. I guess we need to try it to see if it work.

Reply:
@xstefank can you move the files in your github repo to this repo?

Reply:
I would like to first verify that the build will pass. The binary is Linux 64 compatible. Maybe it will need to adjust for the Jenkins environment. But of course, after the job passes, I would like to push everything here.